multisig-wallet/project/contracts/multisig-contract/src/interface.sw

```diff
@@ -1,149 +1,171 @@
 library;
 
-use ::data_structures::signatures::SignatureInfo;
-use ::data_structures::user::User;
+use ::data_structures::{
+    hashing::{
+        ContractCallParams,
+        TransferParams,
+        TypeToHash,
+    },
+    signatures::SignatureInfo,
+    user::User,
+};
 
 abi MultiSignatureWallet {
     /// The constructor initializes the necessary values and unlocks further functionality.
     ///
     /// # Arguments
     ///
-    /// * `users` - The users of the multisig, who can sign transactions to add their approval.
+    /// * `users`: [Vec<User>] - The users of the multisig, who can sign transactions to add their approval.
     ///
-    /// # Reverts
+    /// # Panics
```

Comment:
I've used `Panics` as per https://github.com/FuelLabs/sway-standards/tree/master/standards/src_2 . This PR will remain compliant with SRC-2, but it would be worth bringing up the issue on the standard itself.

Reply:
It's a small amount of work to make this change, so far now I'll comply with SRC-2, and _if_ the standard changes I'll update